lib/features/session/presentation/session_controller.dart

```diff
@@ -40,7 +41,7 @@ class SessionController extends GetxController {
 
   void _goToLogin() {
     _deleteCredential();
-    _clearAllCache();
+    if (!kIsWeb) _clearAllCache();
```

Comment:
Why not clear the cache on the web?

Reply:
it only occurs in web?

Reply:
Follow [https://github.com/hivedb/hive/issues/896](https://github.com/hivedb/hive/issues/896)